sphinxcontrib/needs/needs.py

```diff
@@ -89,7 +90,7 @@
 from sphinxcontrib.needs.roles.need_ref import NeedRef, process_need_ref
 from sphinxcontrib.needs.services.github import GithubService
 from sphinxcontrib.needs.services.manager import ServiceManager
-from sphinxcontrib.needs.utils import INTERNALS, NEEDS_FUNCTIONS
+from sphinxcontrib.needs.utils import INTERNALS, NEEDS_FUNCTIONS, logger
```

Comment:
There gets already a logger imported in this file.
Use this one.

Reply:
Done.